client/ayon_core/hosts/max/plugins/publish/collect_workfile.py

```diff
@@ -55,7 +53,7 @@ def process(self, context):
         }]
 
         instance.data.update(data)
-
+        self.log.info('Collected data: {}'.format(data))
```

Comment:
This should **definitely** not be an info log - and I'd even say that for debug logging just dumping blobs of data is pretty hard to parse.